lib/config/config.js

```diff
@@ -94,12 +94,24 @@ module.exports = function (fs, path, url, convict) {
         format: 'nat',
         env: 'IP_RATE_LIMIT_BAN_DURATION_SECONDS'
       },
+      uidRateLimitIntervalSeconds: {
+        doc: 'Duration of automatic throttling for uids',
+        default: 60 * 15,
+        format: 'nat',
+        env: 'UID_RATE_LIMIT_INTERVAL_SECONDS'
+      },
       uidRateLimitBanDurationSeconds: {
         doc: 'Duration of automatic ban',
         default: 60 * 15,
         format: 'nat',
         env: 'UID_RATE_LIMIT_BAN_DURATION_SECONDS'
       },
+      maxChecksPerUid: {
+        doc: 'Number of checks within uidRateLimitBanDurationSeconds before blocking',
+        default: 100,
+        format: 'nat',
+        env: 'MAX_CHECKS_PER_UID'
+      },
```

Comment:
@larissagaulia right so for above you can do: 

```
uidRateLimit: {
      limitIntervalSeconds: {
        doc: 'Duration of automatic throttling for uids',
        default: 60 * 15,
        format: 'nat',
        env: 'UID_RATE_LIMIT_INTERVAL_SECONDS'
      },
      banDurationSeconds: {
        doc: 'Duration of automatic ban',
        default: 60 * 15,
        format: 'nat',
        env: 'UID_RATE_LIMIT_BAN_DURATION_SECONDS'
      },
      maxChecks: {
        doc: 'Number of checks within uidRateLimitBanDurationSeconds before blocking',
        default: 100,
        format: 'nat',
        env: 'UID_RATE_LIMIT'
      }
}
```

`MAX_CHECKS_PER_UID` -> `UID_RATE_LIMIT`